README.md

```diff
@@ -6,10 +6,7 @@ User Group / Mailing List: [slack channel](https://join.slack.com/t/raftlib/shar
 
 ### Build status
 
-Platform | Status |
----------| -------|
-Linux | [![Build Status](https://travis-ci.org/RaftLib/RaftLib.svg?branch=master)](https://travis-ci.org/RaftLib/RaftLib) 
-Windows | ![CI](https://github.com/f-tischler/RaftLib/workflows/CI/badge.svg?event=push)
+![CI](https://github.com/f-tischler/RaftLib/workflows/CI/badge.svg?event=push)
```

Comment:
Hmm, wondering if these should be set up under the RaftLib org....will it still work with these pointing to your account?

Reply:
Good catch. Github actions are stored in the repository, thus, it should be possible to just edit the link above to point to the original RaftLib repo. Another way would be to merge the PR and then use the context menu in the actions tab to create a new badge link.

Reply:
```suggestion
![CI](https://github.com/RaftLib/RaftLib/workflows/CI/badge.svg?event=push)
```